src/state/connection-mixin.ts

```diff
@@ -42,7 +42,7 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
         language: getLocalLanguage(),
         selectedLanguage: null,
         resources: null as any,
-        localize: () => "",
+        localize: () => "???",
```

Comment:
If the bug is fixed now, I don't think this is needed anymore? All fallbacks in the code will not longer work with this change, and it should not happen. (also could maybe cause some blinking on start?)

Reply:
@bramkragten If it will cause blinking on start, then sure — let's drop this PR. I guess this is because strings load asynchronously, so it's expected for `localize` to return an empty string and then the string will be filled in later when it loads?

Reply:
(Just want to make sure my mental model is correct here. If this line should never be hit, then I would prefer to merge this so that it is obvious if it ever *is* hit.)

Reply:
We set `localize` after we have loaded the translations, so we will hit this when the translations are not loaded yet.